feedhenry/fh-js-sdk
# Render existing repeating sections from existing submissions

# Motivation
https://issues.jboss.org/browse/RHMAP-9288

# Changes
- when opening form with submission (e.g. draft), show all repeating sections in submission
- when opening submission (read-only mode), hide +/- buttons
- expand only first section on every page
- when adding new repeating section, open it

# Verification steps
- draft
  - have a form with repeating section
  - add more sections with some values
  - save it as draft
  - open the draft and check that all the saved values are visible
- submission
  - submit the draft
  - open the submission and check that +/- buttons are not visible and that all the submitted values are visible
